### PR TITLE
FR v2.1-preview.1 update

### PR DESCRIPTION
**Pull request by ramparab**
FRv2.1-preview.1 update
=================
New Model Compose APIs
New [pre-built] Business card APIs
Adding support for Checkbox (aka selectionMark)
Adding support for [Train\Compose] ModelName
Adding Locale parameter to pre-built APIs

### CODE REVIEW

File: specification/cognitiveservices/data-plane/FormRecognizer/readme.go.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-## Go
+ ## Go
```
Comment (kristapratico) [inline specification/cognitiveservices/data-plane/FormRecognizer/readme.go.md:1]: possibly related to the Go CI failure?

File: specification/cognitiveservices/data-plane/FormRecognizer/readme.md [hunk 2]
```diff
@@ -119,4 +119,3 @@ uncomment the  `exclude-file` section below and add the file paths.
 #exclude-file: 
 #  - $(this-folder)/Microsoft.Example/stable/2010-01-01/somefile.json
 ```
-
```
Comment (kristapratico) [inline specification/cognitiveservices/data-plane/FormRecognizer/readme.md:122]: Can't comment on the exact line, but under `# Releases`, let's add:

```
### Release 2.1-preview
These settings apply only when `--tag=release_2_1_preview` is specified on the command line.
``` yaml $(tag) == 'release_2_1_preview'
input-file: 
  - preview/v2.1-preview.1/FormRecognizer.json
```

File: specification/cognitiveservices/data-plane/FormRecognizer/preview/v2.1-preview.1/FormRecognizer.json [hunk 3]
```diff
@@ -0,0 +1,1972 @@
...
+          "$ref": "#/definitions/Confidence"
+        },
+        "fields": {
+          "description": "Dictionary of named field values.",
+          "type": "object",
+          "additionalProperties": {
+            "$ref": "#/definitions/FieldValue"
+          }
+        },
+        "docTypeConfidence": {
```
[the remark this answers is not in the document]
Comment (ramparab) [reply specification/cognitiveservices/data-plane/FormRecognizer/preview/v2.1-preview.1/FormRecognizer.json]: good catch. Fixed.

File: specification/cognitiveservices/data-plane/FormRecognizer/readme.go.md (hunk 1 above)
Comment (ramparab) [reply specification/cognitiveservices/data-plane/FormRecognizer/readme.go.md]: Fixed.

File: specification/cognitiveservices/data-plane/FormRecognizer/readme.md (hunk 2 above)
Comment (ramparab) [reply specification/cognitiveservices/data-plane/FormRecognizer/readme.md:122]: Fixed.

**Pull request merged**